src/man/sssd.conf.5.xml

```diff
@@ -4005,6 +4005,36 @@ subdomain_inherit = ldap_purge_cache_timeout
                             should be comma-separated, such as
                             <quote>enable:passkey, enable:smartcard</quote>
                         </para>
+
+                        <para>
+                            The following table shows which local authentication
+                            methods are currently enabled or disabled for each backend
+                            , with the default local_auth_policy: <quote>match</quote>
+                        </para>
+                        <informaltable frame='all'>
+                        <tgroup cols='4'>
+                        <colspec colname='c1' align='center'/>
+                        <colspec colname='c2' align='center'/>
+                        <colspec colname='c3' align='center'/>
+                        <colspec colname='c4' align='center'/>
+
+                        <thead>
+                        <row><entry namest='c1' nameend='c4' align='center'>
+                            local_auth_policy = match (default)</entry></row>
+                        <row><entry></entry><entry>Passkey</entry>
+                            <entry>Smartcard</entry><entry>2FA</entry></row>
```

Comment:
Hi,

why is there a `2FA` column? This is not controlled by the `local_auth_policy` option.

Additionally I think the wording ` which local authentication methods are currently enabled or disabled for each backend` might be misleading. If e.g. AD is not configured for Smartcard authentication then it will not be enabled on the client if `match` is used.

After reading the existing entry I think an additional sentence to the first paragraph like e.g. "With this option additional methods can be enabled which are evaluated and check locally." would make sense as well.

bye,
Sumit

Reply:
> Hi,
> 
> why is there a `2FA` column? This is not controlled by the `local_auth_policy` option.

Removed.
> 
> Additionally I think the wording ` which local authentication methods are currently enabled or disabled for each backend` might be misleading. If e.g. AD is not configured for Smartcard authentication then it will not be enabled on the client if `match` is used.

I added, `if configured properly, ` to address this.
> 
> After reading the existing entry I think an additional sentence to the first paragraph like e.g. "With this option additional methods can be enabled which are evaluated and check locally." would make sense as well.

Added.